### PR TITLE
fixes to queue jobs properly

### PR DESCRIPTION
**Pull request by lindaxiang**

### CODE REVIEW

File: scripts/scheduler.py [hunk 1]
```diff
@@ -374,6 +375,7 @@ def queue_new_jobs(available_slots, env, config, studies, wes_token, exclude_stu
           try:
               run_info = get_run_state(graphql_url, latest_run_id, wes_token)
               resume = run_info['sessionId'] if run_info.get('sessionId') else False
+              work_dir = run_info['engineParameters']['workDir']
```
Comment (junjun-zhang) [inline scripts/scheduler.py:379]: we could extract `nfs` path from the `workDir` path, then we reassign `nfs` variable here to match want was use for the job in the last run.

Comment (lindaxiang) [reply scripts/scheduler.py:379]: @junjun-zhang but `workflow_version` is also part of the path for `work_dir` and `lauch_dir`.

Comment (junjun-zhang) [reply scripts/scheduler.py:379]: ok, I see. Maybe we can add a new `launch_dir` parameter similar to the newly added `work_dir`?

**Review by junjun-zhang — APPROVED** [review]
looking good, hope resume works properly now

**Pull request merged**